docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,189 @@
+    metadata:
+      name: pmm-token-vmoperator
+      #namespace: default
+    type: Opaque
+    ```
+
+5. Create the Secrets object using the YAML file you created previously. Replace the `<filename>` placeholder with your value.
+
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f pmm-api-vmoperator.yaml -n <namespace>
+    ```
+
+6. Check that the secret is created. The following command checks the secret for the resource named `pmm-token-vmoperator` (as defined in the `metadata.name` option in the secrets file). If you defined another resource name, specify your value.  
+
+   ```{.bash data-prompt="$" }
+   $ kubectl get secret pmm-token-vmoperator -n <namespace>
+   ```
+
+### Create a ConfigMap to mount for `kube-state-metrics` 
+
+The [`kube-state-metrics` (KSM)](https://github.com/kubernetes/kube-state-metrics) is a simple service that listens to the Kubernetes API server and generates metrics about the state of various objects - Pods, Deployments, Services and Custom Resources. 
+
+To define what metrics the `kube-state-metrics` should capture, create the [ConfigMap](https://github.com/kubernetes/kube-state-metrics/blob/main/docs/customresourcestate-metrics.md#configuration) and mount it to a container. 
+
+1. Edit the [example `configmap.yaml` configuration file](https://github.com/Percona-Lab/k8s-monitoring/blob/main/vm-operator-k8s-stack/ksm-configmap.yaml) and specify the `<namespace>`. The Namespace must match the Namespace where you created the Secret. 
+
+2. Apply the configuration
+
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f <github-link> -n <namespace>
+    ```
+
+    As a result, you have the `customresource-config-ksm` ConfigMap created. 
+
+### Install the Victoria Metrics Operator Helm chart
+
+1. Add the dependency repositories of [victoria-metrics-k8s-stack](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-k8s-stack) chart. 
+
+    ```{.bash data-prompt="$" }
+    $ helm repo add grafana https://grafana.github.io/helm-charts
+    $ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    ```
+
+2. Add the Victoria Metrics Operator repository.
+
+    ```{.bash data-prompt="$" }
+    $ helm repo add vm https://victoriametrics.github.io/helm-charts/
+    $ helm repo update
+    ```
+
+3. Update the repositories.
+
+    ```{.bash data-prompt="$" }
+    $ helm repo update
+    ```
+
+4. Modify the default configuration of the `victoria-metrics-k8s-stack` chart. Edit the [`values.yaml`](https://github.com/Percona-Lab/k8s-monitoring/blob/main/vm-operator-k8s-stack/values.yaml) file and specify the following:
+
+    * the IP address / hostname of the PMM server in the `externalVM.write.url` option
+    * the unique name or an ID of the Kubernetes cluster in the `vmagent.spec.externalLabels.k8s_cluster_id` option. Ensure to set different values if you are sending metrics from multiple Kubernetes clusters to the same PMM Server.
+
```

Comment:
*  optionally check the rest of the file and make additional changes, for example, the `prometheus-node-exporter.enabled` to `true` to if you plan to gather metrics for Nodes with the Prometheus node exporter